### DISCUSSION TITLE
idea: Disable auto-complete for login/username boxes.

### DISCUSSION THREAD

**Discussion by SinisterSpatula** [DSJAS/DSJAS#19] [Ideas]
`<input type="text" autocomplete="off">` for login boxes.  especially for baiting we don't want a bunch of names appearing.

**Comment by ejv2** [comment]
Fixed for all built-in themes in [6c6794e1](https://github.com/DSJAS/DSJAS/commit/6c6794e15275155cb5aab46c9c92237fda1ebb01)
